web/src/app/modules/shared/models/form-helper.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { FormHelper } from './form-helper';
+
+describe('FormHelper', () => {
```

Comment:
Since we are testing return value, we can easily add tests for values with `empty string`, `stringified number` and `NaN`.

Reply:
done